plugins/updater/Cargo.toml

```diff
@@ -9,34 +9,42 @@ rust-version = { workspace = true }
 links = "tauri-plugin-updater"
 
 [package.metadata.docs.rs]
-rustc-args = [ "--cfg", "docsrs" ]
-rustdoc-args = [ "--cfg", "docsrs" ]
+rustc-args = ["--cfg", "docsrs"]
+rustdoc-args = ["--cfg", "docsrs"]
 
 [build-dependencies]
-tauri-plugin = { workspace = true, features = [ "build" ] }
+tauri-plugin = { workspace = true, features = ["build"] }
 
 [dependencies]
 tauri = { workspace = true }
 serde = { workspace = true }
 serde_json = { workspace = true }
 thiserror = { workspace = true }
 tokio = "1"
-reqwest = { version = "0.11", default-features = false, features = [ "json", "stream" ] }
+reqwest = { version = "0.11", default-features = false, features = [
+    "json",
+    "stream",
+] }
 url = { workspace = true }
 http = "0.2"
 dirs-next = "2"
 minisign-verify = "0.2"
-time = { version = "0.3", features = [ "parsing", "formatting" ] }
+time = { version = "0.3", features = ["parsing", "formatting"] }
 base64 = "0.21"
 percent-encoding = "2"
-semver = { version = "1", features = [ "serde" ] }
+semver = { version = "1", features = ["serde"] }
 futures-util = "0.3"
 tempfile = "3"
 zip = "0.6"
 tar = "0.4"
 
 [target."cfg(target_os = \"windows\")".dependencies]
 zip = { version = "0.6", default-features = false }
+windows = { version = "0.54", features = [
+    "Win32_System_WinRT",
```

Comment:
this feature shouldn't be needed

Reply:
`HSTRING` is from `Win32_System_WinRT` feature

Reply:
`HSTRING` should be in `windows::core` module and not require any feature flags

Reply:
I found it on https://microsoft.github.io/windows-rs/features/#/0.53.0, didn't know `windows::core::HSTRING` was there

![image](https://github.com/tauri-apps/plugins-workspace/assets/68118705/bb19225e-c1bc-4bc0-87af-633313c93130)

Reply:
that's probably just a re-export, anyways since this PR has changed to use `windows-sys`, it shouldn't matter.